DigitalLearningSolutions.Web/Helpers/RegistrationEmailValidator.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers
+{
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Web.Extensions;
+    using DigitalLearningSolutions.Web.ViewModels.Register;
+    using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+    public static class RegistrationEmailValidator
+    {
+        private const string WrongEmailForCentreErrorMessage =
+            "This email address does not match the one held by the centre";
+
+        private const string EmailInUseErrorMessage =
+            "A user with this email address is already registered; if this is you, " +
+            "please log in at this centre via the My Account page";

```

Comment:
I think it's worth having this, but should be "...log in **and register** at..."
Very rare to have an inactive user account, but if this does happen they just have to talk to HEE.

Reply:
Talked to Steve about this, and he decided we do want the second half of the message. Inactive users seeing this will be a rare occurrence. And I was missing two words which makes it clearer: "please log in and register at this centre via the My Account page"